features/visitor_can_view_articles_in_categories.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Visitor can view articles in categories
+    As a visitor
+    In order to view articles within a specific subject
+    I would like to be able to filter the articles displayed in categories
+
+    Background:
+        Given the following categories exist
+            | name          | id |
+            | Breaking News | 1  |
+            | Politics      | 2  |
+            | Sport         | 3  |
+            | Lifestyle     | 4  |
+            | Health        | 5  |
+
+        Given the following articles exist
+            | title                                 | lead                   | content                                          | category_ids |
```

Comment:
Instead of passing the category id pass in the name.

Reply:
ok changed